src/model/wallet.ts

```diff
@@ -134,18 +252,21 @@ export const signTransaction = async ({ transaction }) => {
 };
 
 export const signMessage = async (
-  message,
+  message: Arrayish | Hexable,
   authenticationPrompt = lang.t('wallet.authenticate.please')
-) => {
+): Promise<null | string> => {
```

Comment:
To be more precise, would this return Hexable (going off of joinSignature on https://docs.ethers.io/v4/api-utils.html#hex-strings)

Reply:
The linter says it returns string and complains if I change it to Hexable instead 

![Screen Shot 2020-08-13 at 4 14 36 PM](https://user-images.githubusercontent.com/1247834/90182560-271d3780-dd80-11ea-97a9-caa871151d62.png)